### ISSUE TITLE
More than 31 days of rewards

### ISSUE THREAD

**Issue by thatguyjordan** [MarkusBordihn/BOs-Daily-Rewards#19]
**Is your feature request related to a problem? Please describe.**

The problem that I see is more of a balance problem in any server/pack that decides that new players could join any month but still wanted to reward longer playtime with increasing tiers of items without breaking balance for new players or giving old players now redundant items. If the mod had a way of instead shifting from the month's items to just a list of consecutive items then server/pack owners would be able to increase the tier of loot with the appropriate intended gametime required. 

**Describe the solution you'd like**

Instead of monthly lists, one singular list could be used instead with the items in that list being able to go past 31 days. So day one for everyone would be the same item no matter what month and day 32 wouldn't be a now redundant tier or inversely to a lower level player, something that should be placed 32 days away.

This would, I imagine, only be really viable in the GUI sense while using compact mode, as to not complicate the page rendering this mode would probably only be compatible with compact rendering 

In this mode it could also be nice to receive the same list of items (eventually) as everyone else regardless of if you miss days/weeks etc

**Describe alternatives you've considered**

- no alts that come to mind

**Additional context**

In short, an example; day 1 on server give iron ingot for example, day 100 give a nether star instead as the player no longer cares for the iron ingot and new players would all follow the same daily rewards regardless of day they start playing

**Comment by MarkusBordihn** [comment]
Thanks for the feedback. What do you think is a good amount of items for this "mode" to define ?

Should be 356 items for 356 days enough or should this be splitted to tiers instead like 3 tier which each 120 items instead ?

It could be also 3 Tier (0-30, 30-60, 60-90) which each 30 items for on quarter of the year, after this the list needs to be reseted instead.

**Comment by thatguyjordan** [comment]
A list that can go 365 days does seem to probably hit the span that I imagine most servers would run for, I imagine many modded forge servers aren't going far over that. However I didn't really necessarily have an expected amount that was the maximum hard limit, but idk how that would change things from the java/playerdata side of things. If a server wanted to ever expand their list then they probably should be able to if it was functionally simple as an implementation

I think in terms of a progression based reward system where the surrounding server might repopulate with new people at any point, taking the people that were there from initial reward implementation would then now go through a bunch of rewards that they may deem useless (after list reset), rather than seeing the day 1 item to day 365+ item progression. 

For some further explanation because idk how it would function code wise, but it's kinda how I imagined it may be added:

If Timmy joins server January 1st and plays every day (with the cool login timer that requires actual playtime there) all the way to December 31st, Timmy deserves some cool item to show that he's hit 365 days in a row. 

If Kevin comes along and joins December 26th because he got a new PC for Christmas, then he'd get the item that Timmy got on January 1st as that is the first in the list.
